src/main/java/CinemaRoomRestService/RestService/domain/entity/CinemaRoom.java

```diff
@@ -0,0 +1,43 @@
+package CinemaRoomRestService.RestService.domain.entity;
+
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.CopyOnWriteArrayList;
+
+@Component
+public class CinemaRoom {
```

Comment:
Your classes is not follows SOLID principles let's talk about that.
Contact me.